Packs/DefaultPlaybook/Playbooks/Default_8_2.yml

```diff
@@ -1102,6 +1071,70 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "70":
+    id: "70"
+    taskid: f2dc11c0-1587-4d62-8c8a-6616dc910ac0
+    type: playbook
+    task:
+      id: f2dc11c0-1587-4d62-8c8a-6616dc910ac0
+      version: -1
+      name: Detonate URL - Generic v1.5
+      playbookName: Detonate URL - Generic v1.5
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
```

Comment:
Please update the URL argument value from URL.Data to URL as defined in the 'Detonate URL - Generic'.